### DISCUSSION TITLE
Product translations

### DISCUSSION THREAD

**Discussion by Robinrrr** [medusajs/medusa#878] [Q&A] [answered]
First of all, medusa looks really cool! :) The only thing I miss in the Admin Area is a product translation option. Is it currently possible to translate products? 

Thanks for your answer 🙌🏼

**Comment by olivermrbl** [comment, marked as answer]
Thanks for the kind words @Robinrrr!

We don't support product translations out of the box. Though, we've helped a couple of customers with translations through a CMS. Our CMS integrations gives you two-way synchronization of variants and products and most CMSs allow you to add multiple locales for your content. Then instead of fetching the products directly from Medusa, you would use the CMS and specify the locale as part of the query :)

We have a guide on setting up Medusa with Contentful [here](https://docs.medusajs.com/how-to/headless-ecommerce-store-with-gatsby-contentful-medusa). Hope this answers your question :)

**Comment by Robinrrr** [comment]
This sounds like a good alternative :)

Thanks, I will try it out
